braintrust/trace/internal/middleware.go

```diff
@@ -33,29 +33,47 @@ func Middleware(router TracerRouter) func(*http.Request, NextMiddleware) (*http.
 		log.Debugf("Middleware: %s %s", req.Method, req.URL.Path)
 		start := time.Now()
 
-		// Intercept the request body so we can parse it and still pass it along.
+		// Determine which tracer to use first
+		var reqTracer MiddlewareTracer
+		if req.URL != nil {
+			reqTracer = router(req.URL.Path)
+		}
+
+		var ctx context.Context
+		var span trace.Span
+		var err error
+
+		if reqTracer == nil {
+			// Unsupported endpoint don't do any tracing
```

Comment:
i might not say "unsupported", just "some endpoints aren't traced. Just pass them along. "